# Microbial community structure of plant-based meat alternatives

Abstract:
A reduction in animal-based diets has driven market demand for alternative meat products, currently raising
a new generation of plant-based meat alternatives (PBMAs). It remains unclear whether these substitutes are
a short-lived trend or become established in the long term. Over the last few years, the trend of increasing
sales and diversifying product range has continued, but publication activities in this field are currently limited
mainly to market research and food technology topics. As their popularity increases, questions emerge about
the safety and nutritional risks of these novel products. Even though all the examined products must be
heated before consumption, consumers lack experience with this type of product and thus further research into
product safety, is desirable. To consider these issues, we examined 32 PBMAs from Austrian supermarkets.
Based on 16S rRNA gene amplicon sequencing, the majority of the products were dominated by lactic acid
bacteria (either Leuconostoc or Latilactobacillus), and generally had low alpha diversity. Pseudomonadota
(like Pseudomonas and Shewanella) dominated the other part of the products. In addition to LABs, a high
diversity of different Bacillus, but also some Enterobacteriaceae and potentially pathogenic species were
isolated with the culturing approach. We assume that especially the dominance of heterofermentative LABs
has high relevance for the product stability and quality with the potential to increase shelf life of the products.
The number of isolated Enterobacteriaceae and potential pathogens were low, but they still demonstrated
that these products are suitable for their presence.

# Comments 

01-metadata contains metadata of the samples and the isolates

02-rawdata contains only the Sanger Sequencing Data of the isolates 16S rRNA gene sequencing data of the samples can be found at ENA under project Accession number PRJEB73555

09-MinIon contains the bash codes for the analysis of the MinIon Data of the tested isolates the MinIon data itself can be found at ENA

A-Z are the R codes used for the analysis
